packages/vuetify/src/components/VTimePicker/VTimePicker.ts

```diff
@@ -197,6 +201,17 @@ export default mixins(
       const value = this.genValue()
       if (value !== null) this.$emit('input', value)
     },
+    emitPicker (value: SelectingTimes) {
+      let val = 'HOUR'
```

Comment:
```suggestion
      let picker = 'HOUR'
```
or maybe even `activePicker`

Reply:
changed to activePicker